prov/efa/src/efa_mr.c

```diff
@@ -176,19 +178,51 @@ struct fi_ops_mr efa_domain_mr_cache_ops = {
 	.regattr = efa_mr_cache_regattr,
 };
 
+static int efa_mr_release(struct efa_mr *mr)
```

Comment:
It seems efa_mr_release() can be merged into efa_mr_close(), and other functions call efa_mr_close(&efa_mr->mr_fid), instead of call efa_mr_release(efa_mr).

Reply:
I see. thank you for trying.